docs/docs/developerguide/README.md

```diff
@@ -218,6 +218,20 @@ email:
     # You will also need to set user and passwordSecret if your
     # mail server requires authentication
 
+privacyPolicy:
+# Contact details of the host and administrator of the teamware instance, if no admin defined, defaults to the host values.
+  host:
+    # Name of the host
+    name: "Service Host"
+    # Host's physical address
+    address: "123 Example Street, City. Country."
+    # A method of contacting the host, field supports HTML for e.g. linking to a form
+    contact: "<a href='mailto:info@examplehost.com'>Email</a>"
+  admin:
+    name: "Service Host"
```

Comment:
Make these example values actually different for host and admin to make the point that they might not be the same person.